Assets/MixedRealityToolkit-SDK/Features/SpatialAwareness/System/Handlers/MixedRealitySpatialAwarenessMeshHandler.cs

```diff
@@ -1,32 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using Microsoft.MixedReality.Toolkit.Core.Definitions.SpatialAwarenessSystem;
 using Microsoft.MixedReality.Toolkit.Core.EventDatum.SpatialAwarenessSystem;
 using Microsoft.MixedReality.Toolkit.Core.Interfaces.SpatialAwarenessSystem.Handlers;
+using Microsoft.MixedReality.Toolkit.Core.Managers;
+using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.SpatialAwarenessSystem.Handlers
 {
     /// <summary>
     /// Class providing the default implementation of the <see cref="IMixedRealitySpatialAwarenessMeshHandler"/> interface.
     /// </summary>
-    public class MixedRealitySpatialAwarehessMeshHandler : IMixedRealitySpatialAwarenessMeshHandler
+    public class MixedRealitySpatialAwarenessMeshHandler : MonoBehaviour, IMixedRealitySpatialAwarenessMeshHandler
```

Comment:
- Is this meant to be a demo script?
    - If not, should we rename it `BaseMixedRealitySpatialAwarenessMeshHandler `? 
- Do we intend to have people inherit from this class or use the interface outright? (I tend to lean for the latter, but if this is in the SDK it's fine to have)
    - If we do intend for people to inherit from this class we should make the interface methods virtual, so they can be overridden.

Reply:
this is going to do a little more than it currently does and i anticipate a significant portion of apps will opt for this basic implementation. the ones that do not are likely to want more control.

I modeled this after the boundary handler.

Reply:
+1 for keeping the class concrete and letting people use the interface if they want more control.

> should we rename it BaseMixedRealitySpatialAwarenessMeshHandler

If we're renaming things we ought to go less verbose, not more. `BaseMixedRealitySpatialAwarenessMeshHandler `is so heavy I can barely parse it. `MixedRealitySpatialAwarenessMeshHandler` is already borderline.

Reply:
@SimonDarksideJ I'm looking at you for our verbose naming conventions 😝 lol

Reply:
this naming convention matches the other feature areas. I am reluctant to change as descriptive is better (MSDN c# coding guidelines state as such, or at least they did last time I looked)